pages/_app.js

```diff
@@ -1,7 +1,12 @@
 import '../styles/globals.css'
+import { OpenAPIProvider } from 'react-openapi-client';
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return (
+    <OpenAPIProvider definition="http://localhost:3000/api/v1/swagger.json">
```

Comment:
Could be better to get the URL from a .env file.

Reply:
I agree it's better if this is in a .env file. 
Also, I think the port should be 3001 (port of frontend) because if we hit the backend directly we will be blocked by CORS policy of the browser. Obviously, we also have to set a rewrite rule in nextjs.

Reply:
both are fixed, ty!!